integration-tests/mysql_test.go

```diff
@@ -0,0 +1,79 @@
+package integration_tests
+
+import (
+	testframework "github.com/cloudfoundry/cloud-service-broker/brokerpaktestframework"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"os"
+	"path/filepath"
+	"regexp"
+	"runtime"
+)
+
+var _ = Describe("Mysql", func() {
+	AfterEach(func() {
+		Expect(mockTerraform.Reset()).NotTo(HaveOccurred())
+	})
+	It("publish mysql in the catalog", func() {
+		catalog, err := broker.Catalog()
+		Expect(err).NotTo(HaveOccurred())
+		service := testframework.FindService(catalog, "csb-google-mysql")
+		Expect(service.Plans).To(HaveLen(3))
+		Expect(service.Tags).To(ContainElement("preview"))
+		Expect(service.Metadata.ImageUrl).NotTo(BeNil())
+	})
+
+	It("should provision small plan", func() {
+		broker.Provision("csb-google-mysql", "small", nil)
+
+		invocations, err := mockTerraform.ApplyInvocations()
+		Expect(err).NotTo(HaveOccurred())
+		Expect(invocations).To(HaveLen(1))
+
+		contents, err := invocations[0].TFVarsContents()
+		Expect(err).NotTo(HaveOccurred())
+		Expect(replaceGUIDs(contents)).To(MatchJSON(getResultContents("mysql-result")))
+	})
+
+	It("user should be able to update database name", func() {
+		broker.Provision("csb-google-mysql", "small", map[string]interface{}{"db_name": "foobar"})
+		Expect(mockTerraform.FirstTerraformInvocationVars()).To(HaveKeyWithValue("db_name", "foobar"))
+	})
+
+	It("user should be able to update database name", func() {
+		broker.Provision("csb-google-mysql", "small", map[string]interface{}{"db_name": "foobar"})
+		Expect(mockTerraform.FirstTerraformInvocationVars()).To(HaveKeyWithValue("db_name", "foobar"))
+	})
+
+	It("user should not be allowed to change mysql cores", func() {
```

Comment:
It looks like the style of testing is different for both tests. Is that intentional? I think it would be nice to have the same style of test for both services. Or maybe will there be a follow up story to backfill them?

Reply:
Yes, it is intentional for now, there where different opinions, the ways of testing could be normalised as we write more tests